onnxruntime/core/framework/op_kernel.cc

```diff
@@ -94,6 +94,18 @@ Status OpKernelContext::GetTempSpaceAllocator(AllocatorPtr* output) const {
   return Status::OK();
 }
 
+Status OpKernelContext::GetCPUAllocator(AllocatorPtr* output) const {
+  // While looking up the allocator from SessionState
+  // (which is called via ExecutionFrame), the allocator lookup
+  // logic doesn't key on OrtAllocatorType, so any OrtAllocatorType
```

Comment:
Would it be better to return the allocator from the CPU EP instead of relying on this internal implementation detail?